appengine/examples.py

```diff
@@ -104,3 +99,74 @@ def get_example(name):
 examples = { 'datalog' : datalog_examples,
              'myrial' : myria_examples,
              'sql' : sql_examples }
+
+
+
+load_twitterk_data = '''T1 = load("s3://uwdb/sampleData/TwitterK.csv",
```

Comment:
For now you should just change the URLs to HTTP (assuming all objects are already public).